webapp/channels/src/components/no_results_indicator/no_results_indicator.tsx

```diff
@@ -31,18 +29,22 @@ interface Props {
 }
 
 const iconMap: {[key in NoResultsVariant]: React.ReactNode } = {
-    [NoResultsVariant.ChannelSearch]: <SearchIcon className='no-results__icon'/>,
-    [NoResultsVariant.Mentions]: <MentionsIcon className='no-results__icon'/>,
-    [NoResultsVariant.FlaggedPosts]: <FlagIcon className='no-results__icon'/>,
-    [NoResultsVariant.PinnedPosts]: <PinIcon className='no-results__icon'/>,
-    [NoResultsVariant.ChannelFiles]: <i className='icon icon-file-text-outline no-results__icon'/>,
-    [NoResultsVariant.ChannelFilesFiltered]: <i className='icon icon-file-text-outline no-results__icon'/>,
-    [NoResultsVariant.UserGroups]: <i className='icon icon-account-multiple-outline no-results__icon'/>,
-    [NoResultsVariant.UserGroupMembers]: <i className='icon icon-account-outline no-results__icon'/>,
-    [NoResultsVariant.UserGroupsArchived]: <i className='icon icon-account-multiple-outline no-results__icon'/>,
+    [NoResultsVariant.Search]: <SearchSVG className='no-results__icon'/>,
+    [NoResultsVariant.ChannelSearch]: <ChannelSearchSVG className='no-results__icon'/>,
+    [NoResultsVariant.Mentions]: <MentionsSVG className='no-results__icon'/>,
+    [NoResultsVariant.FlaggedPosts]: <SavedMessagesSVG className='no-results__icon'/>,
+    [NoResultsVariant.PinnedPosts]: <PinSVG className='no-results__icon'/>,
+    [NoResultsVariant.ChannelFiles]: <ChannelFilesSVG className='no-results__icon'/>,
+    [NoResultsVariant.ChannelFilesFiltered]: <ChannelFilesSVG className='no-results__icon'/>,
+    [NoResultsVariant.UserGroups]: <UserGroupsSVG className='no-results__icon'/>,
+    [NoResultsVariant.UserGroupMembers]: <UserGroupMembersSVG className='no-results__icon'/>,
+    [NoResultsVariant.UserGroupsArchived]: <UserGroupsSVG className='no-results__icon'/>,
 };
 
 const titleMap: {[key in NoResultsVariant]: MessageDescriptor} = {
```

Comment:
Out of the scope, but you would make me a way happier person if you transform this to `defineMessages`.

If not, we can create a help wanted ticket for this.